src/course-unit/course-xblock/CourseXBlock.jsx

```diff
@@ -1,68 +1,104 @@
+      <Card className="mb-1">
+        <Card.Header
+          title={title}
+          actions={(
+            <ActionRow>
+              <IconButton
+                alt={intl.formatMessage(messages.blockAltButtonEdit)}
+                iconAs={EditIcon}
+                size="md"
+                onClick={() => {}}
+              />
+              <Dropdown>
+                <Dropdown.Toggle
+                  id={id}
+                  as={IconButton}
+                  src={MoveVertIcon}
+                  alt={intl.formatMessage(messages.blockActionsDropdownAlt)}
+                  size="sm"
+                  iconAs={Icon}
+                />
+                <Dropdown.Menu>
+                  <Dropdown.Item>
+                    {intl.formatMessage(messages.blockLabelButtonCopy)}
+                  </Dropdown.Item>
+                  <Dropdown.Item onClick={() => unitXBlockActions.handleDuplicate(id)}>
+                    {intl.formatMessage(messages.blockLabelButtonDuplicate)}
+                  </Dropdown.Item>
+                  <Dropdown.Item>
+                    {intl.formatMessage(messages.blockLabelButtonMove)}
+                  </Dropdown.Item>
+                  <Dropdown.Item>
+                    {intl.formatMessage(messages.blockLabelButtonManageAccess)}
+                  </Dropdown.Item>
+                  <Dropdown.Item onClick={openDeleteModal}>
+                    {intl.formatMessage(messages.blockLabelButtonDelete)}
+                  </Dropdown.Item>
+                </Dropdown.Menu>
+              </Dropdown>
+              <DeleteModal
+                category="component"
+                isOpen={isDeleteModalOpen}
+                close={closeDeleteModal}
+                onDeleteSubmit={onXBlockDelete}
               />
-              <Dropdown.Menu>
-                <Dropdown.Item>
-                  {intl.formatMessage(messages.blockLabelButtonCopy)}
-                </Dropdown.Item>
-                <Dropdown.Item>
-                  {intl.formatMessage(messages.blockLabelButtonDuplicate)}
-                </Dropdown.Item>
-                <Dropdown.Item>
-                  {intl.formatMessage(messages.blockLabelButtonMove)}
-                </Dropdown.Item>
-                <Dropdown.Item>
-                  {intl.formatMessage(messages.blockLabelButtonManageAccess)}
-                </Dropdown.Item>
-                <Dropdown.Item>
-                  {intl.formatMessage(messages.blockLabelButtonDelete)}
-                </Dropdown.Item>
-              </Dropdown.Menu>
-            </Dropdown>
-          </ActionRow>
-        )}
-        size="md"
-      />
-      <Card.Section>
-        <div className="w-100 bg-gray-100" style={{ height: 200 }} data-block-id={id} />
-      </Card.Section>
-    </Card>
+            </ActionRow>
+          )}
+          size="md"
+        />
+        <Card.Section>
+          <div className="w-100 bg-gray-100" style={{ height: 200 }} data-block-id={id} />
```

Comment:
`height: 200` please move to the `class`

Reply:
As we decided it before, we leave in the component for simple future refactoring.